packages/wonder-blocks-tooltip/util/ref-tracker.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+/**
+ * This is a little helper that we can use to wrap the react-popper reference
+ * update methods so that we can convert a regular React ref into a DOM node
+ * as react-popper expects, and also ensure we only update react-popper
+ * on actual changes, and not just renders of the same thing.
+ */
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import type {getRefFn} from "../util/types.js";
+
+export default class RefTracker {
```

Comment:
It feels like `forwardRef` from React 16 does something similar.  Do you think we'll be able to use `forwardRef` to replace this in the future?

Reply:
I don't know. It would be nice if `react-popper` had this check built in so we didn't have to worry about it.